<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="eon-swapnili" name="Test" tests="4" failures="1" timestamp="8 Jan 2024 12:48:49 GMT" time="187.64" errors="0">
  <testcase name="TS001_VerifyClickOnShownByExhibitorNameLinkForLineTest" time="25.152" classname="atlantamarket_UAT.GlobalSearch_LineActions"/>
  <testcase name="TS002_VerifyClickOnLocationLinkForLineTest" time="24.24" classname="atlantamarket_UAT.GlobalSearch_LineActions"/>
  <testcase name="TS003_VerifyClickOnMatchingProductsSeeAllLinkForLineTest" time="56.361" classname="atlantamarket_UAT.GlobalSearch_LineActions">
    <failure type="java.lang.AssertionError" message="expected [2] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [401]
	at atlantamarket_UAT.GlobalSearch_LineActions.TS003_VerifyClickOnMatchingProductsSeeAllLinkForLineTest(GlobalSearch_LineActions.java:177)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- TS003_VerifyClickOnMatchingProductsSeeAllLinkForLineTest -->
  <testcase name="TS004_VerifyClickOnTotalProductsSeeAllLinkForLineTest" time="47.304" classname="atlantamarket_UAT.GlobalSearch_LineActions"/>
</testsuite> <!-- Test -->
